Rubberduck.Parsing/VBA/RubberduckParser.cs

```diff
@@ -161,19 +159,55 @@ private void ParseAll()
                 _componentAttributes.Remove(invalidated);
             }
 
-            foreach (var vbComponent in modified)
+            foreach (var vbComponent in toParse)
             {
                 ParseAsync(vbComponent, CancellationToken.None);
             }
         }
 
+        private void AddBuiltInDeclarations(IReadOnlyList<VBProject> projects)
+        {
+            SyncComReferences(projects);
+
+            var finder = new DeclarationFinder(_state.AllDeclarations, new CommentNode[]{}, new IAnnotation[]{});
+            if (finder.MatchName(Tokens.Err).Any(item => item.IsBuiltIn 
+                && item.DeclarationType == DeclarationType.Variable 
+                && item.Accessibility == Accessibility.Global))
+            {
+                return;
+            }
+            
+            var vba = finder.FindProject("VBA");
+            Debug.Assert(vba != null);
+            
+            var errObject = finder.FindClass(vba, "ErrObject", true);
+            Debug.Assert(errObject != null);
+
+            var qualifiedName = new QualifiedModuleName(vba.IdentifierName, vba.IdentifierName, errObject.IdentifierName);
+            var err = new Declaration(new QualifiedMemberName(qualifiedName, Tokens.Err), vba, "Global", errObject.IdentifierName, true, false, Accessibility.Global, DeclarationType.Variable);
+            _state.AddDeclaration(err);
+
+            var debugClassName = new QualifiedModuleName(vba.IdentifierName, vba.IdentifierName, "DebugClass");
```

Comment:
Classes (as well as procedural modules) have their own declarations now. Classes have to have their "IsExposed" attribute set to true since otherwise they are treated as not accessible from outside their projects.

Here's an example of how to instantiate it the new way:

https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Parsing/Symbols/ReferencedDeclarationsCollector.cs#L189

Should have informed you in my last PR about this change, sorry about that.